app/services/participants/sync_dqt_induction_start_date.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Participants
+  class SyncDqtInductionStartDate < BaseService
+    def initialize(dqt_induction_start_date, participant_profile)
+      @dqt_induction_start_date = dqt_induction_start_date
+      @participant_profile = participant_profile
+    end
+
+    def call
+      return false unless FeatureFlag.active?(:cohortless_dashboard)
+      return false if @dqt_induction_start_date.nil?
+      return false if @participant_profile.induction_start_date.present?
+
+      current_induction_record = @participant_profile.induction_records.latest
+      participant_cohort = current_induction_record.cohort
+      dqt_cohort = Cohort.containing_date(@dqt_induction_start_date)
+      return false if dqt_cohort.nil?
```

Comment:
I think that in this case we should also save an error for the participant to track that the target cohort for this participant has not been setup yet in our service. 

This might help to warn us that we need to setup missing cohorts.